lib/passkit/generator.rb

```diff
@@ -75,20 +75,14 @@ def generate_json_pass
       else
         pass[:barcodes] = @pass.barcodes
       end
-
       pass[:appLaunchURL] = @pass.app_launch_url if @pass.app_launch_url
       pass[:associatedStoreIdentifiers] = @pass.associated_store_identifiers unless @pass.associated_store_identifiers.empty?
       pass[:beacons] = @pass.beacons unless @pass.beacons.empty?
-      pass[:boardingPass] = @pass.boarding_pass if @pass.boarding_pass
-      pass[:coupon] = @pass.coupon if @pass.coupon
-      pass[:eventTicket] = @pass.event_ticket if @pass.event_ticket
       pass[:expirationDate] = @pass.expiration_date if @pass.expiration_date
-      pass[:generic] = @pass.generic if @pass.generic
       pass[:groupingIdentifier] = @pass.grouping_identifier if @pass.grouping_identifier
       pass[:nfc] = @pass.nfc if @pass.nfc
       pass[:relevantDate] = @pass.relevant_date if @pass.relevant_date
       pass[:semantics] = @pass.semantics if @pass.semantics
-      pass[:store_card] = @pass.store_card if @pass.store_card
       pass[:userInfo] = @pass.user_info if @pass.user_info
 
       pass[@pass.pass_type] = {
```

Comment:
We can be confident that this does not cause any new problems, because 

 `pass[:coupon]`,  `pass[:eventTicket]`, and `pass[:generic]` were being overwritten by `pass[@pass.pass_type]` here.
 
  `pass[:store_card]`  should've been  `pass[:storeCard]`, so it never did anything useful.